sympy/printing/str.py

```diff
@@ -801,6 +801,9 @@ def sstr(expr, **settings):
 class StrReprPrinter(StrPrinter):
     """(internal) -- see sstrrepr"""
 
+    def _print_Ordinal(self, expr):
```

Comment:
This should rather be in `StrPrinter` which is the general location for string printers in SymPy.

Reply:
(After `_print_Order`.)